src/SFA.DAS.ApplyService.InternalApi/Connected Services/CharityCommissionService/Entities/LatestFiling.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Xml;
+using System.Xml.Serialization;
+
+namespace CharityCommissionService
+{
+    public partial class LatestFiling
+    {
+        [XmlIgnore]
+        public DateTime? AnnualReturnPeriodDateTime
+        {
+            get
+            {
+                try
+                {
+                    return XmlConvert.ToDateTime(this.annualReturnPeriodField, @"dd/MM/yyyy");
+                }
+                catch
```

Comment:
I'm no fan of using catch from programmatic reasons.  If this is the only alternative, then that ok, but if there is a better way to check validity, can you do that instead?